webapp/src/hooks/crud.ts

```diff
@@ -51,7 +51,13 @@ export function useEditPlaybook(id: Playbook['id']): EditPlaybookReturn {
         if (playbook) {
             const updatedPlaybook: PlaybookWithChecklist = {...playbook, ...update};
             setPlaybook(updatedPlaybook);
-            savePlaybook(updatedPlaybook);
+            const save = async () => {
+                savePlaybook(updatedPlaybook).then(() => {
+                    callback?.();
+                });
+            };
+
+            save();
```

Comment:
As-is is just fine, but `async` isn't really needed unless it `await`s something, hence it could be a bit simpler:
```suggestion
            savePlaybook(updatedPlaybook).then(callback);
```